scale/scheduler/threads/recon.py

```diff
@@ -7,8 +7,8 @@
 from scheduler.threads.base_thread import BaseSchedulerThread
 
 
-THROTTLE = datetime.timedelta(seconds=60)
-WARN_THRESHOLD = datetime.timedelta(seconds=1)
+THROTTLE = datetime.timedelta(seconds=1)
```

Comment:
60 seconds to 1 second is correct. I wanted first time reconciliation requests to be submitted quickly, while waiting a minute for subsequent requests for the same task. So the thread itself runs only on a 1 second throttle now, always send first time requests, and waits 60 seconds for resending requests.